src/popup/views/authorize/info/rows/total.jsx

```diff
@@ -9,6 +9,8 @@ class TransactionTotal extends React.Component {
   componentDidUpdate () {
     if (this.noFunds(this.props.tx))
       this.props.dispatch(noFunds('Insufficient funds'));
+    else
+      this.props.dispatch(noFunds(''));
```

Comment:
I would go with `noFunds(false)`. It's more explicit and empty strings as bool values are considered bad pattern.

Reply:
done